libraries/ui/src/CourseCard.tsx

```diff
@@ -136,7 +136,7 @@ export const CourseCard: React.FC<CourseCardProps> = ({
         isEntireCardClickable
         className={clsx(
           'course-card course-card--regular container-lined p-5',
-          'flex flex-col w-[323px] h-[466px] hover:container-elevated',
+          'flex flex-col max-w-full h-[520px] hover:container-elevated',
```

Comment:
Width is now set by the parent, it would still be better not to hardcode the height